src/javascript/binary/websocket_pages/user/account/__tests__/profit_table.js

```diff
@@ -0,0 +1,41 @@
+var expect = require('chai').expect;
```

Comment:
as discussed with @aminmarashi and @jy-binary, we will follow the __tests__ convention of other front-end projects.